bofire/data_models/surrogates/mixed_single_task_gp.py

```diff
@@ -20,6 +25,7 @@ class MixedSingleTaskGPSurrogate(TrainableBotorchSurrogate):
     categorical_kernel: AnyCategoricalKernal = Field(
         default_factory=lambda: HammondDistanceKernel(ard=True)
     )
+    noise_prior: AnyPrior = Field(default_factory=lambda: BOTORCH_NOISE_PRIOR())
```

Comment:
Regarding the test case, you have to update the spec for `MixedSingleTaskGPSurrogate` in `tests/bofire/data_models/specs/surrogates.py` with the new argument, then it should pass the tests.